utilities/decrypt_data/decrypt_multi.py

```
@@ -0,0 +1,237 @@
+        #         "encrypted_col1",
+        #         "encrypted_col2"
+        #     ]
+        # }
+    ],
+    "decryption_api_url": "http://egov-enc-service.egov:8080/egov-enc-service/crypto/v1/_decrypt",
+    "log_dir": "logs"
+}
+
+# ——— Prepare log directory & filename ——————————————————————————————
+os.makedirs(CONFIG["log_dir"], exist_ok=True)
+timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)